### PR TITLE
Adding the demo project video link to the README

### LINKED ISSUE #11: docs: Add the Demo video link in README.md
## Description 
Add the demo video link in the README.md of the project. which is uploaded already in the google drive of girlscript.

### PR DESCRIPTION
**Pull request by SandeepKrSuman**
close #11 

> 
Screenshot (238)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The Project is very beginner friendly to contribute to as well as has a user-fri
 
 ![image](https://user-images.githubusercontent.com/65439761/108522793-3367eb80-72f3-11eb-9ddb-08c49c1549e9.png)
 
+<br>
+
+### Demo Project Video : :tv: https://youtu.be/3dxa599SPT4
```
Comment (khushishikhu) [inline README.md:11]:
```suggestion
### Project's Demo Video : :tv: https://youtu.be/3dxa599SPT4
```

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by khushishikhu** [comment]
@SandeepKrSuman Make this change. Rest is all good. Thank you for your efforts. Keep Coding

**Comment by SandeepKrSuman** [comment]
I have made the changes. Can you please check once and merge the PR...

**Comment by khushishikhu** [comment]
@SandeepKrSuman Thank you. LGTM!!.

**Review by khushishikhu — APPROVED** [review]

**Pull request merged**